src/probnum/diffeq/ode/ivp_examples.py

```diff
@@ -40,10 +40,10 @@ def logistic(timespan, initrv, params=(3.0, 1.0)):
     timespan : (float, float)
         Time span of IVP.
     initrv : RandomVariable,
-        RandomVariable that  describes the belief over the initial
-        value. Usually its distribution is Constant (noise-free)
-        or Normal (noisy). To replicate "classical" initial values
-        use the Constant distribution.
+        Scalar-valued RandomVariable (shape: :code:`()`) that describes the belief
```

Comment:
Just a quick pointer. We recently agreed on a standard on how to document shapes when using type hints. Here's a guide: https://probnum.readthedocs.io/en/latest/development/adding_to_the_api_documentation.html#Docstrings

Reply:
Thanks, I'll adapt the docstrings to this format